plugin/bullets.vim

```diff
@@ -451,7 +445,10 @@ fun! s:insert_new_bullet()
     if l:bullet.text_after_bullet ==# ''
       " We don't want to create a new bullet if the previous one was not used,
       " instead we want to delete the empty bullet - like word processors do
-      call s:delete_empty_bullet(l:curr_line_num)
+      if g:bullets_delete_last_bullet_if_empty
+        call setline(l:curr_line_num, '')
+        let l:send_return = 0
+      endif
```

Comment:
Seemed much cleaner to inline the function rather than pass in multiple parameters.